### PR TITLE
Update the checkbox and clear completed items from the list

### PR DESCRIPTION
**Pull request by trace-kadenyi**
- Update the checkbox through an event listener
- Clear all completed items from the list
- Update the local storage accordingly.
- Update README.md file.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,12 +7,14 @@ addListItem.addEventListener('change', () => {
   const AddedItem = new OneTaskListItem();
   AddedItem.addNewToDo(MyToDo.tasks, addListItem.value);
 });
+// clear completed tasks from list
 const clearSelectedTasks = document.querySelector('.clear');
 clearSelectedTasks.addEventListener('click', () => {
   MyToDo.tasks = MyToDo.tasks.filter((task) => task.completed === false);
   localStorage.setItem('todotasks', JSON.stringify(MyToDo.tasks));
   task.displayToDoList();
 });
```
Comment (codepantha) [inline src/index.js:16]: - [ ] [OPTIONAL] Please update the todo `index `keys when you clear all completed tasks 🙏

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @TracK92 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights ⭐ 
✔️ Descriptive PR
✔️ Good commit history
✔️ Linters are passing
✔️ App working as expected

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (trace-kadenyi) [reply src/index.js:16]: Hi [codepantha](https://github.com/codepantha). Thanks for this suggestion. I will certainly work on it over the weekend. Have a good one.

Comment (codepantha) [reply src/index.js:16]: You are welcome, @TracK92. Enjoy your weekend 👋

**Pull request merged**